controllers/user.go

```diff
@@ -0,0 +1,55 @@
+package controllers
+
+import (
+	"fmt"
+	forms "go-crawler-challenge/forms/user"
```

Comment:
Based on our go package ordering conventions, it should be like this

```go
"fmt"
"net/http"

forms "go-crawler-challenge/forms/user"
...
...
```

Reply:
Changed 👍